README.md

```diff
@@ -0,0 +1,75 @@
+# OCA Addons Repo Template
+
+This is a template created to make easier the task of maintaining OCA addon
+repositories.
+
+## Why?
+
+We have dozens of repos. Most of them look the same, and most of them need
+specific-but-similar configurations for CI, code quality, dependency management, etc.
+
+We need a place where to evolve those things and push them automatically everywhere
+else.
+
+This is that place.
+
+## How to use?
+
+This is a template. It is based on [Copier](https://github.com/pykong/copier), go there
+to read its docs to know how it works.
+
+Quick answer to bootstrap a new repo:
+
+```bash
+# Install copier and pre-commit if missing
+pipx install copier
+pipx install pre-commit
+pipx ensurepath
+# Clone this template and answer its questions
+copier copy https://github.com/OCA/oca-addons-repo-template.git some-repo
+# Commit that
+cd some-repo
+git add .
+pre-commit install
+pre-commit run -a
+git commit -am 'Hello world 🖖'
+```
+
+Quick answer to update a repo:
+
+```bash
+# Update the repo
+cd some-repo
+copier update
+# Reformat updated files
+pre-commit run
+# Commit update
+git commit -am 'Updated from template'
+# Reformat all other files, in case some pre-commit configuration was updated
+pre-commit run -a || git commit -am 'Reformatted after template update'
+```
```

Comment:
To have the possibility to apply this massively to all repos on the 13.0 branch, I think we need a mechanism to opt-out the generation of `.travis.yml` and `README.md`. For the other files it should be ok.

Reply:
Yes, you can add `--skip` to your copier invocation. That's supported.

Reply:
That'd be `--exclude` ? I tested that with `--exclude README.md --exclude .travis.yml` but the it started overwriting my `.git` directory.

Reply:
`--skip` skips the file only if it is present, while `--exclude` totally ignores it. `--skip` would apply in this case.

Also when there is any kind of diff, if ran interactively, copier asks you if you want to skip a file. However `--skip` is still useful to script it.

Reply:
If I read well `--skip` applies to all files? So how to update, say `.pre-commit-config.yaml` while not updating `.travis.yml`?

Reply:
Well, never had to do that, but if I'm not wrong, this should work:

```bash
copier --skip "**" --skip "!.pre-commit-config.yaml" -f
```

Reply:
@yajo At least in the released version of copier that I pipx installed, `--skip` does not take arguments. Is it something new?

Reply:
Humm sorry I was mistaken :speak_no_evil: 

I was confusing it with the `skip_if_exists` option, which is only available through API right now.

Gonna open some issues and work on that.

Reply:
Opened https://github.com/pykong/copier/issues/214 and https://github.com/pykong/copier/issues/215